src/FplBot.Tests/Helpers/Factory.cs

```diff
@@ -40,8 +44,9 @@ private static ServiceProvider BuildServiceProvider(ITestOutputHelper logger)
             {
                 o.Login = configurationSection["Login"];
                 o.Password = configurationSection["Password"];
+                o.NickNames = configurationSection.GetSection("NickNames").Get<List<PlayerNickName>>();
```

Comment:
☝️ Dette funker for testene, men jeg er ikke sikker på om FplbotOptions vil funke ut av boksen i prod. @johnkors?

Reply:
Nicknames på spillere som kommer og går hver sesong føltes som en feil ting å hardkode, så jeg dro det ut i config. Hele greia er jo ganske køddent, så er fremdeles ikke sikker på om det er virkelig verdt noe som helst. Kan godt dra det ut i statisk klasse i stedet, om dere tenker dette er gøyalt nok.

Reply:
Ah, sånn ja. Gjør vel strengt tatt ingenting å ha de hardkodet, så lenge man ikke får inn en ny "Kun" eller "Lord" da :)